kinetic/distribution.yaml

```diff
@@ -3013,6 +3013,17 @@ repositories:
       url: https://github.com/pr2/pr2_common.git
       version: kinetic-devel
     status: maintained
+  pyros_config:
+    doc:
+      type: git
+      url: https://github.com/asmodehn/pyros-config-rosrelease.git
+      version: release/kinetic/{package}
```

Comment:
A repo can contain multiple packages. And in contrast to the release repo they are all together in a single branch.